WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -306,9 +306,20 @@ + (NSDictionary *)processArguments:(XCUIApplication *)app
   [locationManager setPausesLocationUpdatesAutomatically:NO];
   [locationManager startUpdatingLocation];
 
-  CLAuthorizationStatus authStatus = [locationManager respondsToSelector:@selector(authorizationStatus)]
-    ? (CLAuthorizationStatus) [[locationManager performSelector:@selector(authorizationStatus)] integerValue]
-    : [CLLocationManager authorizationStatus];
+
+  CLAuthorizationStatus authStatus;
+  if ([locationManager respondsToSelector:@selector(authorizationStatus)]) {
+    // To fix warning
+    NSInvocation *invocation = [NSInvocation invocationWithMethodSignature:[[locationManager class] instanceMethodSignatureForSelector:@selector(authorizationStatus)]];
+    [invocation setSelector:@selector(authorizationStatus)];
+    [invocation setTarget:locationManager];
+    [invocation invoke];
+    NSInteger status = 0;
```

Comment:
we don't need to have a default value here. Also the variable type might be already set to CLAuthorizationStatus